src/main/kotlin/com/github/mitchellolsthoorn/testgenie/settings/TestGenieSettingsComponent.kt

```diff
@@ -0,0 +1,45 @@
+package com.github.mitchellolsthoorn.testgenie.settings
+
+import com.intellij.ui.components.JBLabel
+import com.intellij.util.ui.FormBuilder
+import javax.swing.JCheckBox
+import javax.swing.JComponent
```

Comment:
Did you found out what was the difference between J and JB UI elements?

Reply:
- From what i can see, JBLabel extends JLabel.
- Another thing is according to the [SDK](https://plugins.jetbrains.com/docs/intellij/misc-swing-components.html), "*the JBSplitter class is JetBrains' replacement for the standard JSplitPane class*", and "*the JBTabs class is JetBrains' implementation of the tab control*".
- As you can also see, they come from different packages (`com.intellij.ui.components` vs `javax.swing`)
- [Swing](https://openjdk.java.net/groups/swing/) is "a Graphical User Interface (GUI) API, offering a set of components that can be used to build rich desktop interfaces" 

So, i guess these are just different implementations of the same component. One is from Java Swing, another is from JetBrains